src/plugins/kibana_react/public/code_editor/code_editor.tsx

```diff
@@ -154,6 +154,32 @@ export class CodeEditor extends React.Component<Props, {}> {
 
     this._editor = editor;
 
+    editor.addCommand(
+      monaco.KeyCode.Escape,
+      () => {
+        editor.trigger('Change tab focus mode', 'editor.action.toggleTabFocusMode');
+
+        const { tabFocusMode } = this._editor.getConfiguration();
+        const ariaLabel = this.props.options && this.props.options.ariaLabel;
+        const extraEditorClassName =
+          this.props.options && this.props.options.extraEditorClassName
+            ? this.props.options.extraEditorClassName
+            : '';
+
+        const ariaTabLabel = tabFocusMode
+          ? 'Tab moves focus. To toggle tab mode, press escape'
+          : 'Tab currently adds tab character. To change tab mode, press escape.';
```

Comment:
```suggestion
          : 'Tab adds tab character. To change tab mode, press escape.';
```

(Just trying to be as to the point as possible.)